src/components/ui/toolbox.ts

```diff
@@ -128,44 +136,21 @@ export default class Toolbox extends EventsDispatcher<ToolboxEventMap> {
     this.api = api;
     this.tools = tools;
     this.i18nLabels = i18nLabels;
+    this.make();
   }
 
   /**
-   * Makes the Toolbox
+   * Returns root block settings element
    */
-  public make(): Element {
-    this.popover = new Popover({
-      scopeElement: this.api.ui.nodes.redactor,
-      searchable: true,
-      messages: {
-        nothingFound: this.i18nLabels.nothingFound,
-        search: this.i18nLabels.filter,
-      },
-      items: this.toolboxItemsToBeDisplayed,
-    });
-
-    this.popover.on(PopoverEvent.Close, this.onPopoverClose);
-
-    /**
-     * Enable tools shortcuts
-     */
-    this.enableShortcuts();
-
-    this.nodes.toolbox = this.popover.getElement();
-    this.nodes.toolbox.classList.add(Toolbox.CSS.toolbox);
-
-    if (import.meta.env.MODE === 'test') {
-      this.nodes.toolbox.setAttribute('data-cy', 'toolbox');
-    }
-
+  public getElement(): HTMLElement {
     return this.nodes.toolbox;
   }
 
   /**
    * Returns true if the Toolbox has the Flipper activated and the Flipper has selected button
    */
   public hasFocus(): boolean | undefined {
-    return this.popover?.hasFocus();
+    return 'hasFocus' in this.popover ? this.popover?.hasFocus() : undefined;
```

Comment:
Same hear, previous version should work just fine

Reply:
there also will be TypeScript error. `hasFocus` exists only in PopoverDesktop and not in PopoverMobile, that's why we need to check weather the property is present